core/src/components/button/button.ios.vars.scss

```diff
@@ -34,7 +34,7 @@ $button-ios-height:                                   3.1em !default;
 $button-ios-border-radius:                            14px !default;
 
 /// @prop - Font size of the button text
-$button-ios-font-size:                                16px !default;
+$button-ios-font-size:                                min(1rem, 22px) !default;
```

Comment:
I updated the max to use the values at `310%` like we discussed. This is now even more of an issue if you have really long buttons. I think it should be fine though because we have the docs that recommend [updating the `white-space` property to `"normal"`](https://ionicframework.com/docs/api/button#overflowing-text-content) and the `min-height` change went out in `7.2`.